i18n/locales/espanol/links.json

```
@@ -2,6 +2,8 @@
   "forum": "https://forum.freecodecamp.org/c/espanol/522",
   "donate": "https://www.freecodecamp.org/espanol/donate/",
   "banner": "https://chat.freecodecamp.org/home",
+  "banner-authenticated": "https://www.freecodecamp.org/espanol/donate",
+  "banner-authenticated-donor": "https://www.freecodecamp.org/espanol/news/como-donar-a-freecodecamp-org-formas-de-dar-y-apoyar-la-mision/",
```

Same suggestion as above with these links:

```suggestion
  "banner": {
    "default": "https://www.freecodecamp.org/",
    "authenticated": "https://www.freecodecamp.org/donate",
    "authenticated-donor": "https://www.freecodecamp.org/news/how-to-donate-to-free-code-camp/"
  },
```